### PR TITLE
refactor Main.rb

### PR DESCRIPTION
**Pull request by edgardsoza**
At this point your codebase is pretty big. The main.rb file may be especially large at this point. Let's try to clean things up!

Make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected. Done
Follow SOLID principles. Done
Pay special attention to the "Single-responsibility Principle". Done
Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit). Done
Extract some pieces of logic into separate class(es). Done

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
-      exit
-    else
-      puts 'The selected option does not exist'
-    end
+  case option
+  when 1
+    app.list_all_books
+  when 2
+    app.list_all_people
+  when 3
```
Comment (AmaduKamara) [inline main.rb:23]: - [x] Good job refactoring your code, but displaying books, and people are not working as it is supposed to. I entered books, and people, and when I entered the option to display any, it showed "No people available" and "No books available". Kindly work on that so your app works fine as expected. 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly work on showing books, and people created and everything else to work as it was before the refactor. 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DeliceLydia — APPROVED** [review]
## Hi @edgardsoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested


### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @edgardsoza 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**